### DISCUSSION TITLE
A user asked me if there's a workflow using IP-Adapter for multi-conditions

### DISCUSSION THREAD

**Discussion by Njasa2k** [Comfy-Org/ComfyUI#1748] [Q&A] [answered]
They are looking for a workflow similar to this
![agfe](https://github.com/comfyanonymous/ComfyUI/assets/99513716/4311203e-33cb-4f49-8740-9e8e3069aafb) 
Image Source: https://ip-adapter.github.io/

Does anyone know of a similar workflow to the above image?

**Comment by jn-jairo** [comment, marked as answer]
### Style
![style](https://github.com/comfyanonymous/ComfyUI/assets/5104869/fd2a31de-4f9a-4319-898f-ff4de7ed0455)

### Pose
![pose](https://github.com/comfyanonymous/ComfyUI/assets/5104869/ad91a562-f47d-44c3-b591-871cf6aeef39)

### Result
![result](https://github.com/comfyanonymous/ComfyUI/assets/5104869/3500fb3f-3e73-444a-9b3e-4cf9165fd790)

The result image contains the workflow.
